packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,173 @@
+        setStoredNotifications(temp_localstorage_data);
+    }, []);
+
+    const countActiveAnnouncements = (): number => {
+        return Object.values(stored_notifications as Record<string, boolean>).reduce(
+            (count: number, value: boolean) => {
+                return value === true ? count + 1 : count;
+            },
+            0
+        );
+    };
+
+    useEffect(() => {
+        countActiveAnnouncements();
+    }, [stored_notifications]);
+
+    const number_ammount_announce = countActiveAnnouncements();
+
+    const handleOnCancel = () => {
+        if (selected_announcement?.switch_tab_on_cancel) {
+            handleTabChange(selected_announcement.switch_tab_on_cancel);
+        }
+        selected_announcement?.onCancel?.();
+        setSelectedAnnouncement(null);
+    };
+
+    const handleOnConfirm = () => {
+        if (selected_announcement?.switch_tab_on_confirm) {
+            handleTabChange(selected_announcement.switch_tab_on_confirm);
+        }
+        selected_announcement?.onConfirm?.();
+        setSelectedAnnouncement(null);
+    };
+
+    const clearNotificationsCallback = () => {
+        const temp_stored_notifications = Object.fromEntries(
+            Object.keys(stored_notifications).map(key => [key, false])
+        );
+        storeDataInLocalStorage(temp_stored_notifications);
+        const temp_notifications = updateNotifications();
+        setStoredNotifications(temp_notifications);
+    };
+
+    return (
+        <div className='announcements'>
+            <button
+                className='announcements__button'
+                onClick={() => setIsOpenAnnounceList(prevState => !prevState)}
+                data-testid='btn-announcements'
+            >
+                <StandaloneBullhornRegularIcon fill='var(--icon-black-plus)' iconSize='sm' />
+                {!is_mobile && (
+                    <Text size='xs' line_height='s' className='announcements__label'>
+                        {localize('Announcements')}
+                    </Text>
+                )}
+                {number_ammount_announce !== 0 && (
+                    <div className='announcements__amount' data-testid='announcements__amount'>
+                        <p>{number_ammount_announce}</p>
+                    </div>
+                )}
+            </button>
+            <div className='notifications__wrapper'>
+                <Announcement
+                    className={clsx('', {
+                        'notifications__wrapper--mobile': is_mobile,
+                        'notifications__wrapper--desktop': !is_mobile,
+                    })}
+                    clearNotificationsCallback={clearNotificationsCallback}
+                    componentConfig={{
+                        clearButtonText: localize('Mark all as read'),
+                        modalTitle: localize('Announcements'),
+                        noNotificationsMessage: localize('No announcements MESSAGE'),
```

Comment:
was `MESSAGE` in uppercase ?

Reply:
I took this as the default value from the example, we can replace it with any string since in our case we never will show it. Do you prefer some other value?

Reply:
lets keep this as an empty string and remove `localize` .
Why should we translate something that is not seen and default ?

Reply:
ok